bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/RetryOptions.java

```diff
@@ -79,85 +77,85 @@
-
+  
+  public static RetryOptions getDefaultOptions() {
+  	return new Builder().build();
+  }
+    
   /**
    * A Builder for ChannelOptions objects.
    */
   public static class Builder {
-    private boolean enableRetries = DEFAULT_ENABLE_GRPC_RETRIES;
-    private int initialBackoffMillis = DEFAULT_INITIAL_BACKOFF_MILLIS;
-    private double backoffMultiplier = DEFAULT_BACKOFF_MULTIPLIER;
-    private int maxElapsedBackoffMillis = DEFAULT_MAX_ELAPSED_BACKOFF_MILLIS;
-    private int streamingBufferSize = DEFAULT_STREAMING_BUFFER_SIZE;
-    private int readPartialRowTimeoutMillis = DEFAULT_READ_PARTIAL_ROW_TIMEOUT_MS;
-    private int maxScanTimeoutRetries = DEFAULT_MAX_SCAN_TIMEOUT_RETRIES;
-    private Set<Status.Code> statusToRetryOn = new HashSet<>(DEFAULT_ENABLE_GRPC_RETRIES_SET);
-    private boolean allowRetriesWithoutTimestamp;
-
+  	private RetryOptions options = new RetryOptions();
+  	
+  	@Deprecated
     public Builder() {
-    }
-
+    	options = new RetryOptions();
+    	options.retriesEnabled = DEFAULT_ENABLE_GRPC_RETRIES;
+    	options.allowRetriesWithoutTimestamp = false;
+    	options.initialBackoffMillis = DEFAULT_INITIAL_BACKOFF_MILLIS;
+    	options.maxElapsedBackoffMillis = DEFAULT_MAX_ELAPSED_BACKOFF_MILLIS;
+    	options.backoffMultiplier = DEFAULT_BACKOFF_MULTIPLIER;
+    	options.streamingBufferSize = DEFAULT_STREAMING_BUFFER_SIZE;
+    	options.readPartialRowTimeoutMillis = DEFAULT_READ_PARTIAL_ROW_TIMEOUT_MS;
+    	options.maxScanTimeoutRetries = DEFAULT_MAX_SCAN_TIMEOUT_RETRIES;
+    	options.statusToRetryOn = new HashSet<>(DEFAULT_ENABLE_GRPC_RETRIES_SET);
+   }
+
+    @Deprecated
     public Builder(RetryOptions options) {
-      this.enableRetries = options.retriesEnabled;
-      this.initialBackoffMillis = options.initialBackoffMillis;
-      this.backoffMultiplier = options.backoffMultiplier;
-      this.maxElapsedBackoffMillis = options.maxElapsedBackoffMillis;
-      this.streamingBufferSize = options.streamingBufferSize;
-      this.readPartialRowTimeoutMillis = options.readPartialRowTimeoutMillis;
-      this.maxScanTimeoutRetries = options.maxScanTimeoutRetries;
-      this.statusToRetryOn = new HashSet<>(options.statusToRetryOn);
-      this.allowRetriesWithoutTimestamp = options.allowRetriesWithoutTimestamp;
+      this.options = options; 
     }
 
     /**
      * Enable or disable retries.
      */
     public Builder setEnableRetries(boolean enabled) {
-      this.enableRetries = enabled;
+    	options.retriesEnabled = enabled;
       return this;
     }
 
     /**
      * Enable or disable retry on deadline exceeded.
      */
-    public Builder setRetryOnDeadlineExceeded(boolean enabled) {
-      if (enabled) {
-        statusToRetryOn.add(Status.Code.DEADLINE_EXCEEDED);
-      } else {
-        statusToRetryOn.remove(Status.Code.DEADLINE_EXCEEDED);
-      }
-      return this;
-    }
+		public Builder setRetryOnDeadlineExceeded(boolean enabled) {
```

Comment:
Please fix the whitespace.  It should remain unchanged.

Reply:
Done.